libs/blocks/caas/utils.js

```diff
@@ -362,7 +366,7 @@ export const getConfig = async (originalState, strs = {}) => {
       eventFilter: state.filterEvent,
       type: state.showFilters ? state.filterLocation : 'left',
       showEmptyFilters: state.filtersShowEmpty,
-      filters: await getFilterArray(state, strs),
+      filters: await getFilterArray(state, strs, country, language),
```

Comment:
`getFilterArray` will take `state`, `country` and `lang` as parameters but you're calling the method with an additional param - `strs`; I don't think this will produce the results you're expecting.

Reply:
Agree, it seems that `strs` wasn't used previously anyway, so it should be safe to simply remove.